homework/src/index.js

```diff
@@ -31,17 +32,151 @@
+      'target': '_blank',
+    });
+
+    const descriptionLine = createAndAppend('tr', table);
+    const descriptionLabel = createAndAppend('td', descriptionLine, {
+      'html': 'Description :',
+      'class': 'label'
+    });
+    const descriptionValue = createAndAppend('td', descriptionLine);
+    const forkLine = createAndAppend('tr', table);
+    createAndAppend('td', forkLine, {
+      'html': 'Forks :',
+      'class': 'label'
+    });
+    const forkValue = createAndAppend('td', forkLine);
+
+    const updateLine = createAndAppend('tr', table);
+    createAndAppend('td', updateLine, {
+      'html': 'Updated :',
+      'class': 'label'
+    });
+    const updateValue = createAndAppend('td', updateLine);
+    const contributorsSec = createAndAppend('section', container, {
+      'class': 'contributors-sec box'
+    });
+    createAndAppend('p', contributorsSec, {
+      html: 'Contributions',
+      'class': 'contributions'
+    });
+    const ul = createAndAppend('ul', contributorsSec);
+
+
     fetchJSON(url, (err, data) => {
-      const root = document.getElementById('root');
       if (err) {
-        createAndAppend('div', root, { html: err.message, class: 'alert-error' });
+        createAndAppend('div', container, {
+          html: err.message,
+          class: 'alert-error'
+        });
       } else {
-        createAndAppend('pre', root, { html: JSON.stringify(data, null, 2) });
+        data.sort((a, b) => a.name.localeCompare(b.name, 'fr', {
+          ignorePunctuation: true
+        }));
+
+        data.forEach((element, i) => {
+          createAndAppend('option', select, {
+            'html': element.name,
+            'value': i
+          });
+        });
+
+        showRepositories(0, data);
+
+        const contributorsInfo = data[0].contributors_url;
+        fetchJSON(contributorsInfo, (err, contributorData) => {
+          showContributors(contributorData, ul);
+        });
+
+        select.addEventListener('change', (e) => {
+          const index = e.target.value;
+          showRepositories(index, data);
+
+          const contributorsInfo = data[index].contributors_url;
+          fetchJSON(contributorsInfo, (err, contributorData) => {
+            ul.innerHTML = '';
+            showContributors(contributorData, ul);
+          });
+
+
+        });
+
       }
+
     });
+    function showRepositories(index, data) {
+      repoNameLink.innerText = data[index].name;
+      repoNameLink.setAttribute('href', data[index].html_url);
+      if (data[index].description === null) {
+        descriptionLabel.innerHTML = '';
+        descriptionValue.innerText = '';
+      } else {
+        descriptionLabel.innerText = "Description :";
+        descriptionValue.innerText = data[index].description;
+      }
+      forkValue.innerText = data[index].forks;
+      const updateRepo = new Date(data[index].updated_at);
+      updateValue.innerText = updateRepo.toLocaleString("en-US")
```

Comment:
Why have you chosen `en-US`? It is better to use the computer default (by leaving out the parameter).